Emitron/Emitron/UI/Settings/SettingsView.swift

```diff
@@ -123,7 +102,7 @@ struct SettingsView: View {
 //  }
 //
 //  static var settingsView: some View {
-//    SettingsView(showLogoutButton: true)
+//    SettingsView()
```

Comment:
Is there a reason this is all commented out?

Reply:
I guess it's commented out because SettingsView_Previews won't load in the Canvas.
Canvas for complex view typically needs some mock properties to generate the Canvas.
So I just fixed the function in the comment because I removed the showLogoutButton property.

Reply:
it seems the only thing stopping it from working was the missing environment Object added SessionController.current now the preview is ok

Reply:
Awesome!